pkg/systemd/services/services.jsx

```diff
@@ -704,8 +704,8 @@ class ServicesPage extends React.Component {
                     if (unit.LoadState == "not-found")
                         return false;
 
-                    if (currentTextFilter && unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter) == -1 &&
-                        unit.Id.indexOf(currentTextFilter) == -1 && !(unit.Names || []).some(name => name.indexOf(currentTypeFilter) != -1))
+                    if (currentTextFilter && !((unit.Description && unit.Description.toLowerCase().indexOf(currentTextFilter.toLowerCase()) != -1) ||
+                        unit_id.indexOf(currentTextFilter) != -1))
```

Comment:
`unit_id` is not the same as `unit.Id`. For example:
`unit_id` for `ModemManager.service` (this is `unit.Id`) is `dus-org.freedesktop.ModemManager1.service`
Also you seem to drop `unit.Names` check ( agree with this, lets filter only by things we show in the UI and not keep some things magically on) but maybe just mention it in the commit messsage?
Plus can we get test for this? (mainly the case insensitive matching for both name and description).

Reply:
This is a deliberate decision. I wrote why in the commit message.